### ISSUE TITLE
Invalid login credentials

### ISSUE THREAD

**Issue by manzazace** [AJTimePyro/Crunchyroll-Account-Checker#24]
Hey, Pyro!

The checker is not working, atleast to me. Yes, I know, that this has been explained in #20 , but even with the explanation and the new checker.py that you've uploaded, it gives me the Invalid login credential result to working accounts (checked manually). Any ideas?

**Comment by AJTimePyro** [comment]
I guess, you have to wait for it, I will try new ways to solve the issue.

**Comment by manzazace** [comment]
that's okay <3

**Comment by JohanChungus** [comment]
> I guess, you have to wait for it, I will try new ways to solve the issue.

Did u solve it?

**Comment by AJTimePyro** [comment]
Nope, I am thinking to close this project because I am unable to solve this issue, hopefully someone else could solve it and make even a better checker tool for crunchyroll.
